candidate.cfg

```diff
@@ -1,5 +1,6 @@
 [buildout]
 extends = buildout.cfg
+primary_indexer_queue_type = REDIS_LIST
```

Comment:
We should use REDIS_LIST for now, but there are faster options for read/write like REDIS_PIPE_EXEC queue type